app/views.py

```diff
@@ -381,12 +382,14 @@ def remove_query(request, qid):
             'message': 'Not logged in or invalid user.'
         }
 
-    return HttpResponse(json.dumps(response), mimetype='application/json')
+    return HttpResponse(json.dumps(response), content_type='application/json')
```

Comment:
```suggestion
    return JsonResponse(response)
```

Reply:
Beyond the scope, [See this](https://github.com/sympy/sympy_gamma/pull/135#issuecomment-612584314)